### PR TITLE
Update preheat kernel section

### PR DESCRIPTION
**Pull request by trungleduc**

### CODE REVIEW

File: content/main.ipynb [hunk 1]
```diff
@@ -176,11 +178,13 @@
     "tags": []
    },
    "source": [
-    "# Templating\n",
+    "# Pre-heated kernels\n",
+    "- It help diminish the waiting time for starting a new python kernel and for executing the requested notebook.\n",
+    "## Entirely pre-executed notebook\n",
+    "<img src=\"full.gif\"  width=\"1000\">\n",
```
Comment (martinRenou) [inline content/main.ipynb]: Have you tried it in Voici? 

I tried adding a local image earlier and Voici was not able to render it. There is probably an issue there. Using an url to the asset published on Github was working though: https://github.com/martinRenou/voici-jupytercon-2023/pull/5

**Review by martinRenou — COMMENTED** [review]
🚀

File: content/main.ipynb (hunk 1 above)
Comment (trungleduc) [reply content/main.ipynb]: indeed, switched to mirror links

**Review by martinRenou — APPROVED** [review]
Thanks!

**Pull request merged**